packages/react-scripts/template/public/robots.txt

```diff
@@ -0,0 +1,2 @@
+User-agent: *
```

Comment:
I'd put a comment link as the very first line

http://www.robotstxt.org/robotstxt.html